src/postgres_deparse.c

```diff
@@ -11869,6 +11872,9 @@ static void deparseStmt(DeparseState *state, Node *node)
 		case T_CommentStmt:
 			deparseCommentStmt(state, castNode(CommentStmt, node));
 			break;
+		case T_CommonTableExpr:
```

Comment:
This should not be needed once we fix the case where we pass `node` instead of `stmt->ctequery` to deparse (when generating truncations).